pkg/cmd/attestation/download/download.go

```diff
@@ -20,7 +20,7 @@ func NewDownloadCmd(f *cmdutil.Factory, runF func(*Options) error) *cobra.Comman
 	opts := &Options{}
 	downloadCmd := &cobra.Command{
 		Use:   "download [<file-path> | oci://<image-uri>] [--owner | --repo]",
-		Args:  cmdutil.MinimumArgs(1, "must specify file path or container image URI, as well as one of --owner or --repo"),
```

Comment:
Pls check this acct as I have never had an account until a few days ago I have been having problems with my apps and files and pics and videos being removed from my device I come to find out it's being done through this app and I have no clue how to use this even pls revert all that's been done off this app